cpp/src/slice2rb/Ruby.cpp

```diff
@@ -257,7 +257,7 @@ Slice::Ruby::compile(const vector<string>& argv)
                         if (!out)
                         {
                             ostringstream oss;
-                            oss << "cannot open`" << file << "': " << IceInternal::errorToString(errno);
+                            oss << "cannot open'" << file << "': " << IceInternal::errorToString(errno);
```

Comment:
This wasn't because of your PR but we're missing a space here:
```suggestion
                            oss << "cannot open '" << file << "': " << IceInternal::errorToString(errno);
```

Reply:
Fixed.